src/xform_to_datalog/datalog_facts_test.cc

```diff
@@ -59,6 +68,8 @@ INSTANTIATE_TEST_SUITE_P(
                       true,
                       "tag")},
                 {}, {}),
+            *(AuthorizationLogicDatalogFacts::create(
+              AuthorizationLogicTest::GetTestDataDir().c_str(), "simple_auth_logic")),
```

Comment:
Similar comment about `c_str` here; use implicit conversion or `native()` or change the signature to remove concern entirely?

Reply:
`::create` takes an `fs::path` now.